packages/bpk-component-checkbox/src/BpkCheckbox.js

```diff
@@ -84,7 +84,7 @@ const BpkCheckbox = props => {
 
 BpkCheckbox.propTypes = {
   name: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```

Comment:
`PropTypes.string` is included in `PropTypes.node` - see the docs here https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes